jfrog-applications/ide/visual-studio-code/extension-settings.md

```diff
@@ -21,6 +21,23 @@ If your JFrog environment is behind an HTTP/S proxy, follow these steps to confi
 
 * Alternatively, you can use the HTTP\_PROXY and HTTPS\_PROXY environment variables.
 
+#### Downloading External Resources Through Artifactory
+
+JFrog VS Code extension required necessary resources for scanning vulnerabilities in your project.
```

Comment:
required --> requires 
...scanning vulnerabilities in your project. --> ...scanning your projects.